persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_37/persist/entities.bal

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
change to license year to 2023 in newly added files

Reply:
updated in [b885ecd](https://github.com/ballerina-platform/persist-tools/pull/140/commits/b885ecdb5655ff525c8b50783b9869fa8df38357)